common/culture/innovations/wc_regional_innovations.txt

```diff
@@ -0,0 +1,32 @@
+#### Local Types ####
+@camel = "gfx/interface/icons/culture_innovations/innovation_camel.dds"
+@elephant = "gfx/interface/icons/culture_innovations/innovation_elephant.dds"
+###Regional Tech
+##Tribal
+innovation_elephantry = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	region = world_innovation_elekk
+	icon = @elephant
+	
+	potential = {
+		##Draenei
+		##this.culture_group = culture_group:draenei_group
+		date = 611.1.1
```

Comment:
There's no such trigger.
Also, the file is in wrong encoding.
```
[15:01:10][lexer.cpp:332]: File 'common/culture/innovations/wc_regional_innovations.txt' should be in utf8-bom encoding (will try to use it anyways)
[15:01:10][pdx_persistent_reader.cpp:229]: Error: "Unknown trigger type: date, near line: 15
" in file: "common/culture/innovations/wc_regional_innovations.txt" near line: 16
```
Please, look for errors in all your files. The error log is in `C:\Users\<User>\Documents\Paradox Interactive\Crusader Kings III\logs\error.log`